src/fuzz_introspector/datatypes/function_profile.py

```diff
@@ -68,3 +69,9 @@ def load_func_branch_profiles(
             bp_loaded[new_branch.branch_pos] = new_branch
 
         return bp_loaded
+
+    def load_func_names(self, input_list: List[str]) -> List[str]:
```

Comment:
Rather than copy and pasting, would it make sense to put this into a helper module (e.g. utils) instead?

Reply:
good call.